examples/flutter_gallery/lib/gallery/backdrop.dart

```diff
@@ -46,7 +46,7 @@ class _IgnorePointerWhileStatusIsNotState extends State<_IgnorePointerWhileStatu
   void initState() {
     super.initState();
     widget.controller.addStatusListener(_handleStatusChange);
-    _ignoring = widget.controller.status != AnimationStatus.completed;
+    _ignoring = widget.status != AnimationStatus.completed;
```

Comment:
The existing code definitely appears to be wrong however I believe this is what's correct (same as _handleStatusChange)

```
 _ignoring = widget.controller.status != widget.status
```